README.md

```diff
@@ -1,2 +1,98 @@
 # osbuilder
+
 Clear Containers guest OS building scripts
+
+## Build guest image ##
+
+### Build a base rootfs ###
+The "rootfs" target will generate a directory
+called "workdir/rootfs".
+
+```
+sudo -E  make rootfs
+```
+
+### (optional) customize clear container rootfs ###
+After generate a initial rootfs (previous step) you can 
+customize it by adding anything you need to rootfs 
+directory.
+
+### Create Clear Container image ###
+The "image" target will take the "workload/rootfs" directory
+and will generate a guest image named container.img in the workdir
+directory compatible with Clear Containers.
+
+```
+sudo -E make image
+```
+
+### Default packages ###
+By default, the rootfs for the Clear Containers image is based on
+Clear Linux for Intel\* Architecture, but WORKLOAD/rootfs could be
+populated with any other source.
+
+The required packages are:
+
+- [systemd]
+- [hyperstart]
+- cc-oci-runtime boot service [cc-agent service]
+- cc-oci-runtime boot target [cc-agent target]
+
+#### Clear Linux based packages security limitations  ####
+
+Clear Linux is not a based rpm Linux distribution and
+the rpm packages are not signed, so there is not way
+to ensure that downloaded packages not trustworthy.
+
+If you are willing to use Clear Linux based images, we encourage you
+to use the Clear Containers images provided from its website
+https://download.clearlinux.org/current/ .
```

Comment:
I don't think you need the space before the full-stop at the end of the sentence.